src/main/java/com/netgrif/application/engine/auth/web/UserController.java

```diff
@@ -174,7 +175,9 @@ public PagedModel<UserResource> getAllWithRole(@RequestBody Set<String> roleIds,
         return resources;
     }
 
-    @PreAuthorize("hasRole('ADMIN')")
+    @Authorizations(value = {
+            @Authorize(authority = "USER_EDIT_ALL", expression = "@userService.isLogged(#userId)")
+    })
     @ApiOperation(value = "Assign role to the user",
             notes = "Caller must have the ADMIN role",
```

Comment:
the `ADMIN` role restriction mentioned in the `notes` has been lifted. This is the case in multiple places in this file